desktop/doctor/src/index.ts

```diff
@@ -256,7 +259,7 @@ export async function runHealthchecks(): Promise<
 
 async function tryExecuteCommand(
   command: string,
-): Promise<HealthchecRunResult> {
+): Promise<HealthCheckRunResult> {
```

Comment:
This is a great catch, too, but it would be ideal if those were individual submissions.